playout/libretime_playout/liquidsoap/1.4/ls_script.liq

```diff
@@ -23,8 +23,9 @@ def create_source()
     l = cue_cut(l)
     l = amplify(1., override="replay_gain", l)
 
-    # the crossfade function controls fade in/out
-    l = crossfade(duration=0., smart=true, l)
+    # The crossfade function controls fade in/out of Tracks and shows
+    # Default values are set in the entrypoint.liq / entrypoint.liq.j2 template
+    l = crossfade(duration=default_crossfade_duration(), fade_in=default_fade_in(), fade_out=default_fade_out(), smart=false, l)
```

Comment:
I tested the PR, and it is not working for me. Also the fade in/out is also broken between tracks inside the same show.

I think the crossfade here happen between the tracks in the queue, but (as explained in the comment before), we use multiple queues, so maybe the crossfade duration here is irrelevant, but the fade in / fade out are.

Reply:
Mhh, I just rested it with a fresh install on Debian 11 and for me it is still working.
Did you change the default values? At my installation the default was "crossfade_duration: 0", "fade_in: 0.5", "fade_out:0.5". With the defaults I did also not here a fading.

When I change them to "crossfade_duration: 3", "fade_in: 1", "fade_out: 1". 
When I have changed them I can hear them fade well.

You might need to restart the server between changing an listening because, as you said in your other comment, there is no trigger.